src/Microsoft.ML.StandardLearners/Standard/LogisticRegression/LogisticRegression.cs

```diff
@@ -109,9 +106,6 @@ internal LogisticRegression(IHostEnvironment env, Arguments args)
         {
             _posWeight = 0;
             ShowTrainingStats = Args.ShowTrainingStats;
-
-            if (ShowTrainingStats && Args.StdComputer == null)
-                Args.StdComputer = new ComputeLRTrainingStdImpl();
```

Comment:
So MKL's Computer is the default value of Args.StdComputer?

Reply:
null is the default. The users will have to opt it to the HalLearners package, to make use of it. 

---
In reply to: [236376849](https://github.com/dotnet/machinelearning/pull/1722#discussion_r236376849) [](ancestors = 236376849)